sorc/build_gdas.sh

```diff
@@ -0,0 +1,33 @@
+#! /usr/bin/env bash
+set -eux
+
+source ./machine-setup.sh > /dev/null 2>&1
+cwd=`pwd`
+
+USE_PREINST_LIBS=${USE_PREINST_LIBS:-"true"}
+if [ $USE_PREINST_LIBS = true ]; then
+  export MOD_PATH=/scratch3/NCEPDEV/nwprod/lib/modulefiles
+else
+  export MOD_PATH=${cwd}/lib/modulefiles
+fi
```

Comment:
I think we removed this `MOD_PATH` business.  We can check w/ @WalterKolczynski-NOAA when he is back.

Reply:
`MOD_PATH` removed